src/main/resources/bidder-config/orbidder.yaml

```diff
@@ -10,3 +10,9 @@ adapters:
         - native
       supported-vendors:
       vendor-id: 559
+    usersync:
+      cookie-family-name: orbidder
+      redirect:
+        url: https://orbidder.otto.de/pbs-usersync?gdpr={{gdpr}}&consent={{gdpr_consent}}&redirect={{redirect_url}}
+        support-cors: false
```

Comment:
Hallo @marki1an, we are Supporting the Orbidder Adapter.
We have a small question, at the "https://orbidder.otto.de/pbs-usersync" Endpoint, we supporting HTTP-OPTION Requests and CORS-Handling.
Can you maybe change the flag to true?

Reply:
I will add this Flag at the Prebid-Server-GO version as well.

Reply:
PR for this behaviour: https://github.com/prebid/prebid-server/pull/3193